lib/open_food_network/permissions.rb

```diff
@@ -183,7 +183,8 @@ def coordinated_order_cycles
 
     def related_enterprises_granting(permission, options = {})
       parent_ids = EnterpriseRelationship.
-        permitting(options[:to] || managed_enterprises).
+        # this options[:to].id needs to be verified/tested
```

Comment:
Should we create an issue for this? It won't happen if it's just a comment.

Reply:
no, I remove that in a later commit. it is tested in many specs, it was just a reminder for me to check it later at that point.